Workflow.md

```diff
@@ -21,7 +21,7 @@ npx aragon ipfs
 ```
 
 
-Then go to the aragon client project and change the `environment.js` file to the following: https://gist.github.com/xseignard/e1952e53e62faa52b4f625344f5fba07
+Then go to the aragon client project and create the `environment.js` file to the following: https://gist.github.com/xseignard/e1952e53e62faa52b4f625344f5fba07
```

Comment:
We have a better PR for this part. I think it still should be change since we are modifying the `environment.js` in `src` folder.

Reply:
I actually had no `environment.js` file in the `aragon` folder

Reply:
And it still worked? @xseignard

Reply:
https://github.com/aragon/aragon/blob/master/src/environment.js